tests/validation/cattlevalidationtest/core/test_native_docker.py

```diff
@@ -112,7 +101,7 @@ def check():
     r_containers = admin_client.list_container(name=name)
     assert len(r_containers) == 1
     c = r_containers[0]
-    c = admin_client.wait_success(c)
+    c = admin_client.wait_success(c, timeout=180)
```

Comment:
LGTM.  Although, I was wondering whether we should have the drone environment pull down the images ahead of time as part of our test framework.  Right now, I don't think we use many different docker images.